apollo-federation/src/query_graph/graph_path.rs

```diff
@@ -271,7 +271,7 @@ impl std::fmt::Display for OpPath {
     fn fmt(&self, f: &mut Formatter<'_>) -> std::fmt::Result {
         for (i, element) in self.0.iter().enumerate() {
             if i > 0 {
-                write!(f, ", ")?;
+                write!(f, "::")?;
```

Comment:
oh, is the delimiter change the one we made for type-conditioned fetching?

Reply:
Not sure tbh, I saw the JS use `::` as a path delimiter somewhere, and it is more readable for both operation paths and fetch paths in my opinion. Note this is only every used for println debugging and is not user-facing